src/jop_tanh.jl

```diff
@@ -0,0 +1,16 @@
+"""
+    F = JopTanh(spc)
+
+where `F` is the hyperbolic tangent operator with domain and range given by `spc::JetSpace`.
+we use 'tanh'(x) = (\exp{x}-\exp{-x})/(\exp{x}+\exp{-x}),
+the derivative of which is 1-tanh(x)^2.
+We expect the domain and range to be real.
+"""
+JopTanh(spc::JetSpace) = JopNl(dom = spc, rng = spc, f! = JopTanh_f!, df! = JopTanh_df!)
+export JopTanh
+
+# TODO: assert denominator in linearization is > 0
```

Comment:
did you mean to include this comment? the linearization does not seem to have a denominator.